content/writing_tests/screenshot.md

```diff
@@ -64,15 +64,122 @@ To reproduce this scenario, you need to capture screenshots of the egg and the p
     try {
         await _navigateTo(url);
         await env.setSimilarity(0.8);
-        await screen.find("source_egg.png").mouseMove().mouseDown(MouseButton.LEFT);
-        await new Region(0, 0, 10, 10).mouseMove();
-        await new Region(500, 700, 50, 100).mouseMove();
-        await screen.find("target_pan.png").mouseMove().mouseUp(MouseButton.LEFT);
+        await screen
+            .find("source_egg.png")
+            .mouseMove()
+            .mouseDown(MouseButton.LEFT)
+            .setX(0)
+            .setY(0)
+            .mouseMove()
+            .move(200, 700)
+            .mouseMove();
+        await screen.find("target_pan.png")
+            .mouseMove()
+            .mouseUp(MouseButton.LEFT);
         await _wait(3000);
     } catch (e) {
         await testCase.handleException(e);
     } finally {
         testCase.saveResult();
     }
-})().then(done);
+})();
+{{< /highlight >}}
+
+## How to Control the Mouse in a Sakuli Test
+
+Before using a `region` it is best to instantiate a `region` over the entire display, to use it multiple times shown in
+the examples above.
```

Comment:
This sentence somehow feels missplaced.

The heading says _"How to Control the Mouse..."_ but does not mention it at all in the first sentence.
Maybe something like

"Sakuli provides two ways to steer mouse movement. Either via coordinates, or via screenshot data. In order to search for image data on your screen you have to instantiate a `Region` spanning the entire display to access it's image content."

will outline the big picture of mouse movement while explaining why a `Region` is required.